io-engine/src/grpc/v1/lvm/mod.rs

```diff
@@ -0,0 +1,40 @@
+use crate::core::{runtime::Runtime, Mthread};
+use once_cell::sync::Lazy;
+use tokio::task::JoinHandle;
+
+pub mod pool;
+pub mod replica;
+
+static RUNTIME: Lazy<Runtime> = Lazy::new(|| {
```

Comment:
What's the idea behind needing an all new runtime - is it to cater someone running two different backends in parallel without stepping on each other's toes?

Reply:
It's because of the ?Send which is added to the trait. We cannot call ?Send futures from the grpc handler which is a Send future.
We already have another runtime but I didn't want to step on its toes as you mentioned.
With latest Rust I think we might get away without this as the macro can generate 2 flavours, Send and ?Send.
I will revisit this once we update the compiler.

Reply:
I don't like the idea of multiple runtimes as well.
Since async traits are already there, can update the compiler first, and make the necessary changes in this PR?

Reply:
I've removed this runtime altogether and increased the max blocking tasks on the existing one.